project.clj

```diff
@@ -6,6 +6,7 @@
    [org.broadinstitute/react-cljs-modal "2017.06.28"]
    [org.clojure/clojure "1.8.0"]
    [org.clojure/clojurescript "1.9.293"]
+   [cljsjs/react-autosuggest "9.0.0-0"]
```

Comment:
A note on this dependency:
I have a PR out against cljsjs to update this to the latest version. In the meantime, it generates a warning in the console. I can't just bring this from NPM because it requires React as a peer dependency, and we don't bring that in from NPM.